server/package.json

```diff
@@ -4,10 +4,15 @@
   "private": true,
   "scripts": {
     "start": "ENV=prod node -r esm app.js",
-    "dev": "source .env && ENV=prod nodemon -r esm app.js",
+    "dev": "source .env && ENV=dev nodemon -r esm app.js",
     "start-w": "set ENV=prod && nodemon -r esm app.js",
     "dev-w": "set ENV=dev && nodemon -r esm app.js",
-    "mongo": "mongo 'mongodb+srv://dpa-jmd2c.gcp.mongodb.net/test'  --username calvin"
+    "mongo:install": "brew update && brew tap mongodb/brew && brew install mongodb-community@4.2",
```

Comment:
Add more script to improve developer's experience:
1. Install mongo Local
2. Run mongo Local/Atlas 
3. Stop Mongo
4. Kill server 8080